src/content/docs/en/guides/fonts.mdx

```diff
@@ -136,14 +129,9 @@ If you are using the [Tailwind integration](/en/guides/integrations-guide/tailwi
 
     Now, all sans-serif text (the default with Tailwind) in your project will use your chosen font and the `font-sans` class will also apply the Inter font.
```

Comment:
So, no objections to the text, but it seems like this example applies the font as your main sans-serif font whereas the earlier examples were targeting individual elements. If that's the case, someone *might* wonder whether you can use this font in Tailwind on a per-element basis, the way we've shown in the other examples, or whether this is only for setting a default. I'm not sure if this is a real concern or not, but came to mind so I'm bringing it up.

Reply:
Tailwind doesn't work on a per-element basis, it gives you a class that you can add to elements. Here we're showing how to change the value of the `font-sans` class, which also happens to be the default (if you assign no classes) in Tailwind.

Reply:
Sounds good to me!